tap_closeio/streams.py

```diff
@@ -165,13 +165,36 @@ def sync_activities(ctx):
     # calls that are in progress _while_ an extraction is happening, no
     # matter the replication frequency or call duration.
     offset_secs = ctx.config.get("activities_window_seconds", (60 * 60 * 24))
+
+    try:
```

Comment:
@harshpatel4crest What happens if someone passes the space `" "`?

Reply:
I can't see and comment/logger realated to it.

Reply:
Default date window of 15 days is used.